src/inmanta/parser/cache.py

```diff
@@ -74,13 +80,22 @@ def un_cache(self, namespace: Namespace, filename: str) -> Optional[List[Stateme
             if not os.path.exists(cache_filename):
                 self.misses += 1
                 return None
+            mtime = os.path.getmtime(filename)
             if os.path.getmtime(filename) > os.path.getmtime(cache_filename):
                 self.misses += 1
                 return None
```

Comment:
Is this still relevant? I think we can just consider the envelope timestamp the authority and ignore the cache mtime altogether.

Reply:
Or is this meant as a performance boost in the most common scenario so we don't need to read the file?